agreement-service.js

```diff
@@ -78,7 +71,3 @@ function factory($http, config) {
 
   return service;
 }
```

Comment:
Our style checkers might be unhappy without a ";" here.  There is one when exporting objects.